### PR TITLE
LPS-152217: Implement NotFoundExceptionMapper

### PR DESCRIPTION
**Pull request by sergiojimcos**
**What is new?**
- Implement `NotFoundExceptionMapper` in `portal-vulcan`
- Update `WikiPageAttachmentResourceImpl` and `CommentReourceImpl`
- Update `WikiPageAttachmentResourceTest` and `CommentReourceTest`

To see this new endpoint you have to follow these steps:

1. Deploy `portal-vulcan-impl`
2. Deploy `headless-delivery-api` and `headless-delivery-impl` in this order.
3. Go to `/o/api`

**JIRA Ticket:**
https://issues.liferay.com/browse/LPS-152217

### CODE REVIEW

**Comment by sergiojimcos** [comment]
ci:test:sf

**Comment by sergiojimcos** [comment]
ci:test:relevant

File: modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/feature/VulcanFeature.java [hunk 1]
```diff
@@ -124,6 +125,7 @@ public boolean configure(FeatureContext featureContext) {
 		featureContext.register(LogContainerRequestFilter.class);
 		featureContext.register(NestedFieldsContainerRequestFilter.class);
 		featureContext.register(NoSuchModelExceptionMapper.class);
+		featureContext.register(NotFoundExceptionMapper.class);
```
Comment (ccorreagg) [inline modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/feature/VulcanFeature.java:128]: This is not sorted properly

File: modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/CommentResourceImpl.java [hunk 2]
```diff
@@ -89,8 +89,20 @@ public void deleteComment(Long commentId) throws Exception {
 				String externalReferenceCode)
 		throws Exception {
 
-		BlogsEntry blogsEntry = _getBlogsEntry(
-			blogPostingExternalReferenceCode, siteId);
+		BlogsEntry blogsEntry;
+
+		try {
+			blogsEntry = _getBlogsEntry(
```
Comment (ccorreagg) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/CommentResourceImpl.java:95]: This method is already throwing a NotFoundException

**Comment by sergiojimcos** [comment]
ci:test:sf

**Comment by sergiojimcos** [comment]
ci:test:relevant

**Comment by sergiojimcos** [comment]
ci:test:relevant

**Comment by sergiojimcos** [comment]
ci:forward

**Pull request closed without merging**